frontend/packages/console-app/src/components/quick-starts/controller/QuickStartContent.tsx

```diff
@@ -51,7 +51,7 @@ const QuickStartContent: React.FC<QuickStartContentProps> = ({
           tasks={tasks}
           conclusion={conclusion}
           allTaskStatuses={allTaskStatuses}
-          nextQuickStart={nextQuickStart}
+          nextQuickStart={nextQuickStart[0]}
```

Comment:
`nextQuickStart` could also be undefined in some cases where a quick start doesn't define any next quick start. Better to add a null check.

Reply:
added